src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -179,7 +186,41 @@ describe('Visual refresh toolbar only', () => {
         await expect(page.isClickable(findDrawerById(wrapper, 'security')!.toSelector())).resolves.toBe(false);
         await expect(page.isClickable(findDrawerById(wrapper, 'circle-global')!.toSelector())).resolves.toBe(true);
         await expect(page.isClickable(findDrawerById(wrapper, 'circle2-global')!.toSelector())).resolves.toBe(true);
-      });
-    });
+      })
+    );
   });
+
+  test(
+    'should prevent the horizontal page scroll from appearing during resize',
+    setupTest(async page => {
+      const hasHorizontalScroll = () =>
+        page.getBrowser().execute(() => document.body.scrollWidth - document.body.clientWidth > 0);
```

Comment:
NB, we have another similar check here: https://github.com/cloudscape-design/components/blob/50a9c9e9bd86ddddb072be2fc87e2950ea190f68/src/app-layout/__integ__/mobile.test.ts#L52-L53

After merging this, we should follow up and unify detection logic. (Discussed it DM, your version is better, because it properly counts for the scrollbar widths)